aspnet/conceptual-overview/understanding-aspnet5-apps.rst

```diff
@@ -36,192 +29,142 @@ has been refactored into files that are generally smaller and more focused.
-
-By default, the bower task is executed using gulp, which is configured in *gulpfile.js*. The current web template’s gulpfile includes tasks for copying and cleaning script and CSS files from the bower folder to a ``/lib`` folder in ``wwwroot``.
-
-.. image:: understanding-aspnet5-apps/_static/bower-npm-folders.png
+Dependency are then further configured in their own section within each *bower.json* file, indicating how it should be deployed to the *wwwroot* folder.
 
 .. _`server side dependencies` :
 
 Server Side Dependency Management
 ---------------------------------
 
-The *References* folder details the server-side references for the project. It should be familiar to ASP.NET developers, but it has been modified to differentiate between references for different framework targets, such as the full ASP.NET 5.0 vs. ASP.NET Core 5.0.  Within each framework target, you will find individual references, with icons indicating whether the reference is to an assembly, a NuGet package, or a project. Note that these dependencies are checked at compile time, with missing dependencies downloaded from the configured NuGet package source (specified under Options – NuGet Package Manager – Package Sources).
+The *References* folder, shown within **Solution Explorer** in Visual Studio, details the server-side references for the project. It should be familiar to ASP.NET developers, but it has been modified to differentiate between references for different framework targets, such as the full DNX 4.5.1 vs. DNX Core 5.0.  Within each framework target, you will find individual references, with icons indicating whether the reference is to an assembly, a NuGet package, or a project. Note that these dependencies are checked at compile time, with missing dependencies downloaded from the configured NuGet package source (specified under **Options** –> **NuGet Package Manager** –> **Package Sources**).
 
 .. image:: understanding-aspnet5-apps/_static/references.png
 
+For more information about NuGet, see `Introduction to NuGet <https://docs.asp.net/en/latest/conceptual-overview/nuget.html>`_.
+
 Configuring the Application
 ---------------------------
 
 ASP.NET 5 no longer stores configuration settings in XML files (*web.config* and *machine.config*). Configuration is now stored in *appsettings.json*, which was designed specifically for storing app configuration settings. The default ASP.NET project template includes Entity Framework, and so specifies the database connection string details in the *appsettings.json* file included in the project.
 
-.. code-block:: JSON
-
-	{
-		"Data": {
-			"DefaultConnection": "Server=(localdb)\\mssqllocaldb;Database=aspnet5-FundamentalConcepts"
-		},
-
-		"Entity": {
-			"ApplicationDbContext": {
-				"ConnectionString": "Data:DefaultConnection:ConnectionString"
-			}
-		}
-	}
+.. literalinclude:: understanding-aspnet5-apps/sample/src/WebApplication1/appsettings.json
```

Comment:
Also show how these values are accessed in the `ConfigureServices` method in Startup.cs:

```c#
  services.AddEntityFramework()
    .AddSqlServer()
    .AddDbContext<ApplicationDbContext>(options =>
      options.UseSqlServer(Configuration["Data:DefaultConnection:ConnectionString"]));
```